module/product/tests/Domain/ValueObject/SkuTest.php

```diff
@@ -29,6 +29,14 @@ public function testGetValue(string $sku): void
         $this->assertEquals($sku, $sku->getValue());
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     */
+    public function testInvalidVAlue(): void
+    {
+        $sku = new Sku('yKL3rWXluEM7NapnMSVpYHpicQJkpJ0Obfx91mma0xwnQxUfsrwy5Nfki1LUZR4qYolBTlUFDO4RkeINIkjPzMfTSit0bQZJevXA6GMsj0LnSZaiT1bBfr00vtKWqLAPollonRzb6GBVlT5U9I6nC49r3Vnj2jUgpA73CvfnVnFBNnHqCaI2Cu48SKaVSRGgROhoD1dGPvvq98okavZ3ktVHk0TcyyiyfoH52U3gP3J5bNTVZngivjPJAqtOW8TO');
```

Comment:
This entry will be incompatible with the currently implemented formatting rules. Maximum 120 characters per line.